spec/observables/dom/ajax-spec.ts

```diff
@@ -298,39 +294,41 @@ describe('ajax', () => {
 
     MockXMLHttpRequest.mostRecent.respondWith({
       status: 200,
-      contentType: 'application/json',
```

Comment:
`contentType` does nothing for this test. Here we assume that the browser is working as expected, and we deal with the `responseType`, which the browser sets for us. Since all of that process is mocked, switching to using `responseType` only in our tests.